src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,39 @@
+.nav {
+  &__list {
+    @extend %list-reset;
+    display: flex;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    @extend %link-reset;
+    display: block;
+    text-transform: uppercase;
+    font-weight: 500;
+    transition: color 0.3s;
```

Comment:
let's be consistent about transitions duration and use one duration value for every transition on the page